### DISCUSSION TITLE
Watchman oil tank monitor _NOT_ sonic

### DISCUSSION THREAD

**Discussion by s0x00c** [merbanan/rtl_433#2373] [Q&A]
Hey

I bought a DVB stick to try and monitor my watchman oil tank monitor (NOT sonic version).
I started the monitoring but nothing came through.
After that I decided to increase the verbosity level to "-vv"
and now I can see this output:

```
[Protocols] Registered 204 out of 237 device decoding protocols
[SDR] Found 1 device(s)
[SDR] trying device  0:  Realtek, RTL2838UHIDIR, SN: 00000001
Detached kernel driver
Found Fitipower FC0012 tuner
[SDR] Using device 0: Generic RTL2832U OEM
Exact sample rate is: 250000.000414 Hz
[SDR] Sample rate set to 250000 S/s.
[Input] Bit detection level set to 0.0 (Auto).
[SDR] Tuner gain set to Auto.
[Input] Reading samples in async mode...
[SDR] Tuned to 433.920MHz.
Allocating 15 zero-copy buffers
[Baseband] low pass filter for 250000 Hz at cutoff 25000 Hz, 40.0 us
[markisol_decode] bits_per_row[0] = 1
[pulse_slicer_osv1] preamble 1  886 822
[philips_aj7010_decode] wrong number of bits (325)
[ert_idm_decode] rows=1, row0 len=650
[ert_netidm_decode] rows=1, row0 len=650
[markisol_decode] bits_per_row[0] = 0
[markisol_decode] bits_per_row[0] = 0
[markisol_decode] bits_per_row[0] = 0
[markisol_decode] bits_per_row[0] = 0
.
.
.
[markisol_decode] bits_per_row[0] = 1
[markisol_decode] bits_per_row[0] = 1
[markisol_decode] bits_per_row[0] = 1
```
Does this mean my Watchman oil tank monitor is not supported yet, since it can't decode the data it's receiving?

**Comment by zuckschwerdt** [comment]
Hard to say. You will need to find and grab a sample https://triq.org/rtl_433/ANALYZE.html
(transmissions might only be every hour or so unless you enter "rebinding" / sync mode, the magnet procedure)
Then inspect the sample file or upload as zip here.

**Comment by s0x00c** [comment]
Hey, I am monitoring now with "-S unkown"
I got some packets already, but I *think* the most interesting packets captures will be the ones attached:
[packets.zip](https://github.com/merbanan/rtl_433/files/10697136/packets.zip)

**Reply by zuckschwerdt** [reply]
The first three are empty, the last one looks like LoRA maybe. drop on https://triq.org/pdv/ to check.

**Reply by s0x00c** [reply]
Hey, thanks for the suggestion, but I really don't understand what I am seeing there.
What should I be looking for? I don't really understand what all these settings mean.
![image](https://user-images.githubusercontent.com/91124858/217830186-1bb9249a-29f5-4c46-8e2b-6952e248ff24.png)

**Comment by zuckschwerdt** [comment]
Look at the spectrogram above. It should look like one of these files: https://github.com/merbanan/rtl_433_tests/tree/master/tests/oil_watchman

**Reply by s0x00c** [reply]
I've been capturing since 3 hours now and these empty packets as you call them, show up, but then these packets with size over 1 mb show up as well.
They do not look like the one you suggested https://github.com/merbanan/rtl_433_tests/tree/master/tests/oil_watchman/05)/g668_433.92M_1000k.cu8 but there is clearly a signal detected there. 
The signal appears to be pretty regular as well. with in 3 hours or a bit more I have 4 of these packets, that could be 'hourly'.
Now, to decode the signal itself, I have no clue where to start.
I'll add the packets here for reference: 
[packets.zip](https://github.com/merbanan/rtl_433/files/10698707/packets.zip)

Would you need more packets captured to decode this or what way can I help ?

**Reply by s0x00c** [reply]
If I zoom in on the spectrum analyzer, I can see this:
![image](https://user-images.githubusercontent.com/91124858/217857755-c7fab887-96c5-4d59-bec8-d9e48b8bf034.png)
Bright 'dashes' but they all seem to have the same length which is not what I expect if I am looking for a digital signal consisting of 0's and 1's.
Could it be that I need to monitor a broader spectrum? Meaning 433.2 to 434.5 or something like that ? Is that even possible?

**Reply by zuckschwerdt** [reply]
Yes, there does not appear to be any data. It's nearly hourly transmissions of 2.35 seconds of warmup/preamble only. It's not legal to transmit that long. Strange.
The frequencies is correct, no need to adjust.

**Reply by s0x00c** [reply]
Hey


[packets.zip](https://github.com/merbanan/rtl_433/files/10704800/packets.zip)
I've been capturing all night, as I said I would, but I am afraid the pattern is just repeating itself constantly. (every hour'ish, not exactly but close enough)
one thing that did stand out was this morning, I think I captured someones garage port opener or remote control device.

![image](https://user-images.githubusercontent.com/91124858/218030984-d457ec9f-69ec-47c7-b869-f67c2570bb5e.png)
But that really doesn't help my case in particular, just 'interesting'

**Reply by zuckschwerdt** [reply]
Seems like the sensor is broken then. Perhaps try a hard reset by taking out batteries. Or try the rebinding, move magnet close for some time (procedure should be in some manual).

**Reply by s0x00c** [reply]
The sensor is definitely not broken since the receiver in the house still updates without issue. 
I pulled it from the socket yesterday to check if there were any marking stating it was working on a different frequency, but it's actually noted right in the plastic mould that it 433.92Mhz. 
I will try the magnet method when I get a chance and find the exact procedure.
Thanks for the support thus far.

**Comment by zuckschwerdt** [comment]
Oh? The display unit still receives the sensor? Then maybe do full recording (Gigabytes needed) at the expected time. Use `-w filename.cu8 -T 10m` to record 10 minutes. (300MB). Maybe something is in there but not "seen" by rtl_433.

**Reply by s0x00c** [reply]
I did this, with a 20 minute interval. It gave me a file of nearly 600 Mb.
I've compressed it to 470 Mb and put it on 'wetransfer' since file size is limited here to 25 mb
I did execute with '-g 60' option as @merbanan suggested and '-vv'
Please find the link here: https://we.tl/t-ewhvyQ6LUt
![image](https://user-images.githubusercontent.com/91124858/218082000-91822aaf-b291-4441-8503-78c79474aa29.png)

**Reply by zuckschwerdt** [reply]
The band looks completly quiet expect for that one transmission in the middle. To extract it
`dd if=file.cu8 of=watchman_433.92M_250k.cu8 skip=287 bs=1m count=2`

And again there is no data in there. Sorry, no clue what's going on. If the "real" transmission is somewhere else try a wider band (`-s 1024k`) or shift the frequency around and look for other signals.

**Reply by s0x00c** [reply]
I did extract the data that you provided a dd command for and it ends up looking exactly the same as before (I've added to the post for completeness)
[watchman_433.92M_250k.zip](https://github.com/merbanan/rtl_433/files/10707092/watchman_433.92M_250k.zip)
I'll see what I can do with the -s 1024k that you suggest

**Comment by merbanan** [comment]
"Found Fitipower FC0012 tuner", this tuner needs a fixed gain setting to work at its fullest. Try tuning the gain.

**Reply by s0x00c** [reply]
Do you by any chance know the gain that I should configure?
Which setting is normal for this tuner?

**Comment by merbanan** [comment]
Try max, -g 60 will give that.
